test/test_helper.exs

```diff
@@ -10,8 +10,8 @@ ExUnit.start()
 
 IO.puts "[mongod v#{version}]"
 
-{_, 0} = System.cmd("mongo", ~w'mongodb_test --eval db.dropDatabase() --port 27001')
-{_, 0} = System.cmd("mongo", ~w'mongodb_test2 --eval db.dropDatabase() --port 27001')
+{_, 0} = System.cmd("mongo", ~w'mongodb_test --eval db.dropDatabase() --port 27017')
+{_, 0} = System.cmd("mongo", ~w'mongodb_test2 --eval db.dropDatabase() --port 27017')
```

Comment:
The ports being 27001 was actually here because of the three mongo instances that get started automatically when running the test suite (See `start_mongo.bash`).

I will need to see if these lines are still needed. I also wanted to move the starting of mongo inside this file so that it will automatically close upon test completion instead of keeping the three extra servers around.